central/splunk/violations_test.go

```diff
@@ -1331,12 +1330,9 @@ func (s *violationsTestSuite) TestResponseContentType() {
 }
 
 func (s *violationsTestSuite) TestViolationsHandlerError() {
```

Comment:
I was trying to remember what this test does and could not. I suggest removing it.

The idea is that http handler intercepts panics and aborts the response but this test does not seem to validate that. I suggest just removing it.